examples/model_compress/QAT_torch_quantizer.py

```diff
@@ -35,6 +35,7 @@ def train(model, quantizer, device, train_loader, optimizer):
         loss = F.nll_loss(output, target)
         loss.backward()
         optimizer.step()
+        quantizer.update_step()
```

Comment:
i think it is a little strange to call both `optimizer.step()` and `quantizer.update_step()`.

If `update_step()` is called every time `optimizer.step()` is called, then i think the api `update_step()` can be deleted.

Reply:
add patch_optimizer in quantizer.